.ci/releasePipeline.groovy

```diff
@@ -141,3 +127,175 @@ pipeline {
+                              secretValues: [[envVar: 'repository', vaultKey: 'client_repository_url ', isRequired: true]]]]
+    ) {
+        if (!fileExists('sign-client.jar')) {
+            sh '''
+                curl --request GET $repository/testautomation-release-local/com/dynatrace/services/signservice/client/1.0.[RELEASE]/client-1.0.[RELEASE].jar
+                     --show-error
+                     --silent
+                     --globoff
+                     --fail-with-body
+                     --output sign-client.jar
+               '''.replaceAll("\n", " ").replaceAll(/ +/, " ")
+        }
+    }
+}
+
+void buildBinary(Map args = [command: null, version: null, dest: null]) {
+    stage('🏁 Build ' + args.command) {
+        sh "make ${args.command} VERSION=${args.version} OUTPUT=${args.dest}"
+    }
+}
+
+void signWinBinaries(Map args = [source: null, version: null, destDir: null, projectName: null]) {
+    stage('Sign binaries') {
+        signToSignService(source: args.source, version: args.version, destDir: args.destDir, signAction: "SIGN", projectName: args.projectName)
+    }
+}
+
+void createShaSum(Map args = [source: null, dest: null]) {
+    stage('Compute SHA sum') {
+        sh "sha256sum ${args.source} | sed 's, .*/, ,' > ${args.dest}"
+    }
+}
+
+void signShaSum(Map args = [source: null, version: null, destDir: null, projectName: null]) {
+    stage('Sign SHA sum') {
+        signToSignService(source: args.source, version: args.version, destDir: args.destDir, signAction: "ENVELOPE", projectName: args.projectName)
+        sh "cat ${args.source}"
+    }
+}
+
+void createTarArchive(Map args = [source: null, dest: null]) {
+    stage('Create tar archive') {
+        tar(dir: args.source, file: args.dest)
+    }
+}
+
+def pushToDynatraceStorage(Map args = [name: null, source: null, version: null]) {
+    stage('Deliver tar to storage') {
+        withEnv(["name=${args.name}", "version=${args.version}", "source=${args.source}",]) {
+            withVault(vaultSecrets: [[path        : 'keptn-jenkins/monaco/artifact-storage-deploy',
+                                      secretValues: [
+                                          [envVar: 'repo_path', vaultKey: 'repo_path', isRequired: true],
+                                          [envVar: 'username', vaultKey: 'username', isRequired: true],
+                                          [envVar: 'password', vaultKey: 'password', isRequired: true]]]]
+            ) {
+                sh '''
+                    curl --request PUT $repo_path/monaco/$version/$name
+                         --user "$username":"$password"
+                         --upload-file $source
+                         --fail-with-body
+                   '''.replaceAll("\n", " ").replaceAll(/ +/, " ")
+            }
+        }
+    }
+}
+
+void signToSignService(Map args = [source: null, version: null, destDir: null, signAction: null, projectName: null]) {
```

Comment:
nit: 

```suggestion
void signWithSignService(Map args = [source: null, version: null, destDir: null, signAction: null, projectName: null]) {
```

reads more clearly/correct to me

Reply:
done